### PR TITLE
Adding RxCocoa reactive extension badgeColor on UITabBarItem 

### PR DESCRIPTION
**Pull request by LucianoPAlmeida**
Adding RxCocoa reactive extension badgeColor on UITabBarItem  available for iOS 10+ and tvOS 10+.

### CODE REVIEW

**Comment by freak4pc** [comment]
Hey @LucianoPAlmeida ! Thanks for the addition and willingness to help. 🙏 

In general, if you'll look at other similar PRs - a decision was made to not add additional RxCocoa extensions that are trivial to create (e.g. simple Binders) because it creates a heavier code base that needs more maintenance and provides value only to a small portion of users. 

When the addition is not trivial (e.g. some fancy DelegateProxy of a popular delegate that's hard to get right), it might be worth adding it to RxCocoa core, but otherwise - these simpler ones might be better suited as a PR to RxCocoa. 

Let me know what you think :)
Shai.

**Comment by LucianoPAlmeida** [comment]
Hey @freak4pc :)
Is very common to create specific custom binders, this one thought could be useful for other users. 
Thank you for the feedback and I totally understand the point 👍 
Feel free to close this one :))

**Comment by LucianoPAlmeida** [comment]
> When the addition is not trivial (e.g. some fancy DelegateProxy of a popular delegate that's hard to get right), it might be worth adding it to RxCocoa core, but otherwise - these simpler ones might be better suited as a PR to RxCocoa.

Sorry to ask but I'm a bit confused 🙃. RxCocoa Core is different from RxCocoa? 😆

**Comment by freak4pc** [comment]
@LucianoPAlmeida - Sorry! I was distracted, I meant RxSwiftExt could be a good place for it :) 
Thanks for your understanding!

File: Tests/RxCocoaTests/UITabBarItem+RxTests.swift [hunk 1]
```diff
@@ -15,17 +15,36 @@ final class UITabBarItemTests : RxTest {
 }
 
 extension UITabBarItemTests {
-    func testBadgetValue_Text() {
```
Comment (freak4pc) [inline Tests/RxCocoaTests/UITabBarItem+RxTests.swift]: If you have a second to make a PR just for this fix that would be awesome - typos in code are a bummer 😂

Comment (LucianoPAlmeida) [reply Tests/RxCocoaTests/UITabBarItem+RxTests.swift]: For sure 😄

**Pull request closed without merging**